bin/filewatcher

```diff
@@ -54,7 +54,7 @@ def restart(pid, env, cmd)
   begin
     raise Errno::ESRCH unless pid
 
-    Process.kill(9, pid)
+    Process.kill('TERM', pid)
```

Comment:
Seems like the term signal could be trapped and the process may never exit right? Do we need to have a wait for graceful exit timeout and then kill -9 if it takes too long to die?

Reply:
> Seems like the term signal could be trapped and the process may never exit right?

Yes.

Any signal can be trapped except `KILL`.

> Do we need to have a wait for graceful exit timeout and then kill -9 if it takes too long to die?

I thought about it. But I don't know how long. Option? With which default? I guess it should be resolved after problematic cases.

Reply:
> Option?

Sure. I was originally proposing the kill signal as a new option as well. 

> With which default?

My first thought is that a few seconds should be fine. Hopefully most processes would exit properly and it won't come up

Reply:
I don't want to implement something in what we don't need guaranteed.

If you want to test this for your case — let's do it and find out any problems, if not — I'm going to wait new problems, if they will be here at all (I'm not sure about this).